### DISCUSSION TITLE
Increase User Interactivity

### DISCUSSION THREAD

**Discussion by dh4340** [cmu-sei/GHOSTS#436] [Ideas]
**Increase User Interactivity**

Enhance user interactions to make them more realistic, such as:
	* Randomly clicking on links or opening attachments
	* Configurable levels of interactivity (e.g., decreasing frequency of clicks over time)
	* Possible timeline chart

**Comment by sei-dupdyke** [comment]
This should be possible by just decreasing the wait times on looped activity, no?

**Reply by dh4340** [reply]
Yes, but this doesn't offer an easily configurable user interface to enable realistic user generation. What I'm suggesting is the creation of an element of automated, controllable unpredictability
